### PR TITLE
Mo 86 employee category field

### PR DESCRIPTION
**Pull request by RajatMCAF**
When checking this branch, it is required to uninstall and install custom HR timesheet module

### CODE REVIEW

File: mo_hr_timesheets/data/hr_demo.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        </record>
+
+        <record id="hr.contract_type_temporary" model="hr.contract.type">
+            <field name="name">Part-time</field>
+        </record>
+
+        <record id="hr.contract_type_interim" model="hr.contract.type">
+            <field name="name">Supply</field>
+        </record>
+        <record id="contract_type_term" model="hr.contract.type">
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/data/hr_demo.xml:14]: remo it plz

File: mo_hr_timesheets/views/hr_contract_views.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <field name="model">hr.contract</field>
+        <field name="inherit_id" ref="hr_contract.hr_contract_view_form"/>
+        <field name="priority" eval="16"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='contract_type_id'][@groups='!hr_contract.group_hr_contract_manager']"
+                   position="replace"/>
+            <xpath expr="//field[@name='contract_type_id'][@groups='hr_contract.group_hr_contract_manager']"
+                   position="replace"/>
+
+            <xpath expr="//field[@name='hr_responsible_id']"
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/views/hr_contract_views.xml:14]: scr
change here as well plz

File: mo_hr_timesheets/data/hr_demo.xml (hunk 1 above)
Comment (RajatMCAF) [reply mo_hr_timesheets/data/hr_demo.xml:14]: Do you want me to remove Supply employee category?

Comment (MORPSmcaf) [reply mo_hr_timesheets/data/hr_demo.xml:14]: ok you can leave it

File: mo_hr_timesheets/views/hr_contract_views.xml (hunk 2 above)
Comment (RajatMCAF) [reply mo_hr_timesheets/views/hr_contract_views.xml:14]: Do you want me to remove breadcrumbs for this view?

File: mo_hr_timesheets/views/hr_contract_views.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+                   position="before">
+                <field name="contract_type_id" options="{'no_create': True, 'no_open': True}"/>
```
Comment (MORPSmcaf) [inline mo_hr_timesheets/views/hr_contract_views.xml:16]: make Employee Category required

Comment (RajatMCAF) [reply mo_hr_timesheets/views/hr_contract_views.xml:16]: Done

**Review by MORPSmcaf — APPROVED** [review]

**Pull request merged**